File: Web/RMA-Web-Shell/projects/admin/src/app/configuration-manager/shared/commission-band.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class CommissionBand extends BaseClass {
  commissionBandId: number;
  commissionBandName: string;
  minSalaryBand: number;
  maxSalaryBand: number;
  commissionRate: number;
}

File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/CommissionBand.cs

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class CommissionBand
    {
        public int CommissionBandId { get; set; } // CommissionBandId (Primary key)
        public string CommissionBandName { get; set; } // CommissionBandName (length: 50)
        public decimal MinSalaryBand { get; set; } // MinSalaryBand
        public decimal MaxSalaryBand { get; set; } // MaxSalaryBand
        public decimal CommissionRate { get; set; } // CommissionRate
        public bool IsDeleted { get; set; } // IsDeleted
        public string CreatedBy { get; set; } // CreatedBy (length: 50)
        public System.DateTime CreatedDate { get; set; } // CreatedDate
        public string ModifiedBy { get; set; } // ModifiedBy (length: 50)
        public System.DateTime ModifiedDate { get; set; } // ModifiedDate
    }
}
